src/main/java/net/datafaker/service/files/EnFile.java

```diff
@@ -147,6 +147,7 @@ public String getPath() {
         "lorem.yml",
 //            "lovecraft.yml",
 //            "markdown.yml",
+        "marvel_snap.yml",
```

Comment:
Why is marvel_snap added here? I don't think that's needed.

Reply:
correct, I'll remove it.